src/pages/Overview/FiltersAndSorts.ts

```diff
@@ -53,13 +53,19 @@ export namespace FiltersAndSorts {
       isNumeric: false,
       param: 'h',
       compare: (a: NamespaceInfo, b: NamespaceInfo) => {
-        let diff = b.inError.length - a.inError.length;
-        if (diff !== 0) {
-          return diff;
-        }
-        diff = b.inWarning.length - a.inWarning.length;
-        if (diff !== 0) {
-          return diff;
+        if (a.status && b.status) {
+          let diff = b.status.inError.length - a.status.inError.length;
+          if (diff !== 0) {
+            return diff;
+          }
+          diff = b.status.inWarning.length - a.status.inWarning.length;
+          if (diff !== 0) {
+            return diff;
+          }
+        } else if (a.status) {
+          return 1;
+        } else if (b.status) {
+          return -1;
```

Comment:
So... items whose health is not yet fetched always go first.
Here, I would prefer to show, first, items with errors/warnings; then, items whose health is not fetched yet OR with green health (i.e., give same priority and order by name). Just because, as is, you can get this weird (but funny) behavior where cards start moving like a train:

![peek 2019-01-24 18-13](https://user-images.githubusercontent.com/23639005/51716774-d7431800-2003-11e9-882a-c08d14f8c1a6.gif)

Reply:
Oh, I *never* know when I have to return positive or negative :-)
I'll do the contrary!

Reply:
done